### PR TITLE
Update tutorial-web-app-dotnet-sign-in-users.md

### PR DESCRIPTION
**Pull request by faulycoelho**
add missing package Microsoft.Identity.Web.DownstreamApi

### CODE REVIEW

**Comment by Court72** [comment]
@cilwerner

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/identity-platform/tutorial-web-app-dotnet-sign-in-users.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Identity related **NuGet packages** must be installed in the project for authent
 
 1. In the top menu of Visual Studio, select **Tools > NuGet Package Manager > Manage NuGet Packages for Solution**.
 1. With the **Browse** tab selected, search for and select **Microsoft.Identity.Web.UI**. Select the **Project** checkbox, and then select **Install**.
+1. With the **Browse** tab selected, search for and select **Microsoft.Identity.Web.DownstreamApi**. Select the **Project** checkbox, and then select **Install**.
```
Comment (cilwerner) [inline docs/identity-platform/tutorial-web-app-dotnet-sign-in-users.md:38]:
```suggestion
1. With the **Browse** tab selected, search for and select **Microsoft.Identity.Web.UI**. Select the **Project** checkbox, and then select **Install**. Repeat this for **Microsoft.Identity.Web.DownstreamApi**.
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
A minor modification from me. Thanks

**Comment by Court72** [comment]
Thanks @cilwerner! I committed your suggestion.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

**Comment by cilwerner** [comment]
#sign-off

**Pull request merged**